Assets/Scripts/DepositTextAnimator.cs

```diff
@@ -0,0 +1,43 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
+
+namespace Filibusters
+{
+    public class DepositTextAnimator : MonoBehaviour 
+    {
+        private Text mDepositText;
+        [SerializeField]
+        private int mNewSize;
+        private int mOriginalSize;
+        private int mSizeDiff;
+
+        [SerializeField]
+        private int mPlayerNum;
+
+        void Start() 
+        {
+            mDepositText = GetComponent<Text>();
+            mOriginalSize = mDepositText.fontSize;
+            mSizeDiff = mNewSize - mOriginalSize;
+            EventSystem.OnCoinDepositedEvent += (int ownerId, int newDepositBalance) => 
+            {
+                if (NetworkManager.GetPlayerNumber(PhotonPlayer.Find(ownerId)) == mPlayerNum)
+                {
+                    StartCoroutine(AnimateText());
+                }
+            };
+        }
+        
+        IEnumerator AnimateText()
+        {
+            for (int i = 1; i <= 5; i++)
+            {
+                // I originally meant to divide by 5 here, however, this provides a better effect
+                mDepositText.fontSize = mOriginalSize + mSizeDiff * (i / 2);
+                yield return new WaitForSeconds(0.05f);
```

Comment:
for the lerp, change this to a WaitForFrame or whatever return just returns till the next frame and then lerp the growth based on the movement (or possibly use an animation curve if you don't feel like lerping!!)